venues/ICLR.cc/2018/Conference/webfield/conferenceWebfield_tabs.js

```diff
@@ -89,11 +89,15 @@ function load() {
       );
     });
 
-    authorNotesP = Webfield.get('/notes/search', {
-      term: user.id,
-      group: CONFERENCE,
-      content: 'authors',
-      source: 'forum'
+    authorNotesP = Webfield.get('/user/profile', {
+      email: user.id
```

Comment:
what happens if the user.id is a tilde id?

Reply:
is it possible for user.id to be a tilde id? I thought it was the login of the user, which should be an email address

Reply:
Ohh I see, it is the info of the logged user, you are right